src/span_context.js

```diff
@@ -159,7 +128,7 @@ export default class SpanContext {
             return null;
         }
 
-        // todo(oibe) is it better to throw an exception here, or return null?
+        // TODO(oibe) is it better to throw an exception here, or return null?
```

Comment:
we must always avoid exceptions, not only because they bubble up and can crash the app, but also because they confuse Node's optimizer